app/code/Transoft/Blog/Controller/Adminhtml/Post/Index.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Controller\Adminhtml\Post;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\View\Result\PageFactory;
+
+/**
+ * Class Index
+ *
+ * Index controller class
+ */
+class Index extends Action implements HttpGetActionInterface
+{
+    /**
+     * @var bool|PageFactory
+     */
+    protected $resultPageFactory;
```

Comment:
Why is it **protected**? Please refer to [2.7 Class design](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#2-class-design). Please fix **all** cases.

Reply:
fixed